### ISSUE TITLE
Add "AMERICAN REVOLUTIONARY GEOGRAPHIES ONLINE" next to logo in top banner

### ISSUE THREAD

**Issue by alexandergknoll** [bplmaps/argo-astro#83]
How does everyone feel about adding this to the top banner, similarly to how we have it in the footer?  I think it might provide some immediate and valuable context for what ARGO stands for.  Mockup:

![localhost_4321_](https://github.com/bplmaps/argo-astro/assets/9630849/f8e515e1-e30c-49ba-bab1-92f30be2d81f)

**Comment by hkemp2128** [comment]
I feel good about that! Agreed about the immediate and valuable context

**Comment by garrettdashnelson** [comment]
@almontg Yeah, I think if the graphic designer can get a version as a full logo lockup, that'd be great. With that said, I actually really like the elegance of the acronym-only version.

**Comment by almontg** [comment]
@garrettdashnelson @alexandergknoll we have a version from the graphic designer with the full name on the bottom, which I'll send over (via email to Garrett) for now -- they're a bit swamped at the moment but at least we can work this version into the footer
